lte/protos/session_manager.proto

```diff
@@ -540,3 +542,255 @@ service CentralSessionController {
+//}
+//enum BearerOperationCode {
+//   CREATE_NEW_EPS_BEARER = 0;
+//   DELETE_EXISTING_EPS_BEARER = 1;
+//   MODIFY_EXISTING_EPS_BEARER = 2;
+//}
+//enum EpsParameterIdentity {
+//    MAPPED_EPS_QOS_PARAMETERS = 0;
+//    MAPPED_EXTENDED_EPS_QOS_PARAMETERS =1;
+//    TRAFFIC_FLOW_TEMPLATE = 2;
+//    APN_AMBR = 3;
+//    EXTENDED_APN_AMBR = 4;
+//}
+//ExtProtocolConfigOption
+//message ExtProtocolConfigOption {
+//   repeated uint32 protocolid = 0;
+//   repeated int32 length_of_protocol_content = 1;
+//   repeated string protocol_content = 2;
+//   repeated uint32 containerid = 3;
+//   repeated int32 length_of_container_content = 4;
+//   repeated string container_content = 5;
+//}
+//LadnServiceArea
+
+message LadnServiceArea {
+   string dnn_value = 1;
+   repeated TrackingAreaIdentityList identity_list = 2;
+}
+message TrackingAreaIdentityList {
+   repeated TypeOfList type_of_list = 1;
+   repeated int32 number_of_elements = 2;
+   string MCC=3;
+   string MNC=4;
+   string TAC= 5;          // tac is havaing length 8 bytes,MCC is 3 bytes, MNC is 3 bytes
+}
+enum TypeOfList  {
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_NON_CONSECUTIVE_TAC_VALUES = 0;
+   LIST_OF_TACS_BELONGING_TO_ONE_PLMN_WITH_CONSECUTIVE_TAC_VALUES = 1;
+   LIST_OF_TAIS_BELONGING_TO_DIFFERENT_PLMNS = 2;
+}
+enum priorityaccess  {
+   High = 0;
+   Normal = 1;
+   Low = 2;
+}
+
+enum SMSessionFSMState {
+        CREATING_0 = 0;
+        CREATE_1 = 1;
+        ACTIVE_2 = 2;
+        INACTIVE_3 =3;
+        RELEASED_4 = 4;
+}
+
+// Common message used in both 4G and 5G the naming convention of 4G and 5G are normalized.
+
+//message M5GCommonSessionContext {
+//    RATType rat_type = 1; //Define enum TS-29.571 - section 5.4.3 structure Imported from session_manager.proto file.
+//    string SubscriberID = 2; //Length 15 bytes, SubscriberID is IMSI in 4G and SUPI in 5G
+//    RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+//    SMSessionFSMState fsm_session_state = 4;//PDU session state to mirror with SessionD
+//    uint32 fsm_version = 5;
+//    string apn = 6;//variable length,  contains a APN, defined in 3GPP TS 23.003 [12].
+//    string imei = 7; // pei - permanent equipment ID. Variable length of 16 bytes. TS-23.003 - 6.4 Like IMEI or IMEISV
+//    string user_location_info = 8; //up to 6 byte. TS-23.003 - 28.15.3
+//}
+
+//PDU session Establishment from AMF to SMF
+//SET MESSSAGE
+message M5GSMSessionContext {
+  bytes pdu_session_id = 1;   //One byte
+  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
+  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  priorityaccess priority_access = 4;
+  AccessType access_type = 5;     //Define enum TS-29.571 - section 5.4.3
+  SMSessionFSMState fsm_session_state = 6; //PDU session state to mirror with SessionD
+  SubscriberID sid = 25;
```

Comment:
SubscriberID is included in CommonContext

Reply:
Yes, its part of CommonContext. Taken out